tests/web/test_web_about_openstax_tutor.py

```diff
@@ -0,0 +1,165 @@
+
```

Comment:
Missing the file docstring and imports:
`NameError: name 'test_case' is not defined`
`NameError: name 'web' is not defined`
etc.

Reply:
Updated